### ISSUE TITLE
Events view as tiles

### ISSUE THREAD

**Issue by stursberg** [hpi-swt2/sport-portal#380]
**This depends on #379 for tile component**
**#374 depends on this for event logic**

New Layout for "Events" Page:

Header and footer are out of scope.
If the event has a event-picture, then tile has this picture. If not, then we use 1 of 2 default pictures (they can be easily found in Flexibus channel)

In wide-screens every tile has a 25% of width with paddings/margins.
If width for content is <=800px, then it's only 2 blocks in a row.
In it is <=400 px, then only 1 block in a row. (IT IS ONLY EXAMPLE - the most important thing, that it looks good on every screen 300 to 1600px).
Don't forget that content has max-width:1200px!

- 'Deine Events' = I'm a member, my team is a member, I created the event
- 'Deine Events' is not shown when there is no data

![event1.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46ed829f2d96d2ebc3abaf51ab57f5973d047d317c05f3782387e72aba1579545f594eac42515a10adfbd82db4aa2e32db025fb28c341521842f1bbeb98b15be04bb26894f82b7b58f4ec1ed2f68.png)

![event2.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46ed829f2d96d1ebc3abaf51ab57f5973d047d317c05f3782387e72aba1579545f594eac42515a10adfbd82db4aa2e32db025fb28c341521842f1bbeb98515be04bb26894f82b4bc8a44c4eb2f68.png)

**Comment by jkbe** [comment]
**Estimation: L**

**Comment by jcremerius** [comment]
@zhamanakov Should the user be able to click on the event to reach the event specific site? If yes where should the user click to go to the even?
Thanks!

**Comment by jkbe** [comment]
Yesterday in the sprint planning we agreed that the whole tile should be clickable.

**Comment by jkbe** [comment]
It seems to me like right now there is no image displayed in the frontend for each event. However, it seems the logic for that was already implemented in #294 @nikriek?! Just to make sure we don't do work twice...

**Comment by stursberg** [comment]
@jcremerius yes and all the tile is a link (pic + name + all params)

**Comment by nikriek** [comment]
I do the upload and model related stuff. You do the actual displaying off tiles. Ok?

**Comment by stursberg** [comment]
- [x] remove 70% width from `.container`, content should be as wide as header (max 1200px)
- [x] break description onto new line 
- [x] maybe decrease font-size for description (e.g. to 0.9rem) so that each description line doesn't take up more than one line 
- [x] remove link-underlining from description 

- [ ] truncate headlines dynamically 
- [ ] Anmeldung bis and Startdatum onto separate lines without | in between 

- [x] make picture height dynamic, just crop pictures of different dimensions into that frame which always has the same height-width ratio 
- [x] switch to 2 column layout at 900px 
- [x] Deine Events and Alle Events should be h2 (or h4 like on the teams index page)
- [x ] less whitespace between Events and Alle Events if user is logged out

**Comment by emanuel-metzenthin** [comment]
Most PO requests have been fixed in #379

**Comment by stursberg** [comment]
Default picture is now way too big (1,7 Mb).
It should be max. 100 Kb.

Custom pictures should be also compress to max 100 Kb.

**Comment by nikriek** [comment]
https://github.com/hpi-swt2/sport-portal/pull/521

**Comment by stursberg** [comment]
Why contains every tile "Max. Spieleranzahl" and not "Anmeldung: bis XXXX   |  Start: XXXX"?

![Screen Shot 2018-02-04 at 13.11.38.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f76b7191a379ec0c2a50415c46be5d4a1f06a3a3dc78e2a436628f0d0beed23f0b26933a43a9b8841fbf02b325df1e87e7ba9850d2aa6c7bb3d4d216744b470a83ed5c3daa984ec95ff14fe74e52a0153462f43b.png)

**Comment by jcremerius** [comment]
As issue 379 is not done yet, the event tiles are not up to date on dev. When 379 is approved, we can open a pull request and merge to dev. On our branch, the tiles look different. Is it okay to have Startdatum and Anmeldedatum in two rows, becasue they take too much space?
It would look like this.
![unbenannt](http://user-images.githubusercontent.com/15142695/35779405-b472b3e2-09cc-11e8-9d8a-2be001cba5c1.png)

Thank you!

**Comment by jkbe** [comment]
I just enabled automatic deployment of your branch to sport-portal-pr on Heroku as we have that deployment available again and it seems @zhamanakov is eager to give feedback. ;) Or would you like to have the deployment for feedback on #379 @emanuel-metzenthin @MariusDanner?

**Comment by jcremerius** [comment]
Here is the link: https://sport-portal-pr.herokuapp.com

**Comment by stursberg** [comment]
@jcremerius yes, 2 lines for Startdatum and Anmeldedatum.
But name should be in max 1 line.
